static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -171,12 +171,14 @@ function useReplayData({replaySlug, orgSlug}: Options): Result {
         return [];
       }
 
+      const finishedAtClone = new Date(replayRecord.finishedAt);
+      finishedAtClone.setSeconds(finishedAtClone.getSeconds() + 1);
```

Comment:
I manually tested that `setSeconds` will roll-over the `minute` counter if needed.